src/components/ui/toaster.tsx

```diff
@@ -9,27 +9,27 @@ import {
   ToastViewport
 } from '@/components/ui/toast';
 import useToast from '@/hooks/useToast';
-import { AlertCircle, Ban, CheckCircle, Info, icons } from 'lucide-react';
+import { AlertCircle, Ban, CheckCircle2, Info } from 'lucide-react';
 
 const IconToast = {
   info: {
     icon: <Info className="text-primary-blue" />,
-    color: 'text-primary-blue',
+    color: 'text-black',
     borderColor: 'border-primary-blue'
   },
   warning: {
     icon: <AlertCircle className="text-yellow-400" />,
-    color: 'text-yellow-400',
+    color: 'text-black',
     borderColor: 'border-yellow-400'
   },
   error: {
     icon: <Ban className="text-red-500" />,
-    color: 'text-red-500',
+    color: 'text-black',
     borderColor: 'border-red-500'
   },
   success: {
-    icon: <CheckCircle className="text-green-500" />,
-    color: 'text-green-500',
+    icon: <CheckCircle2 className="text-green-500" />,
+    color: 'text-black',
```

Comment:
Don't use color key, if they are all the same

Reply:
check it please a7e5f31b2f348a1658f0d0e4550c03080b40f9f8